man/man1/pkcsicsf.1.in

```diff
@@ -68,6 +68,21 @@ the opencryptoki config file containing token configuration information
 .IP "/etc/opencryptoki/\fItoken_name.conf\fP"
 contains ICSF configuration information for the ICSF token
 
+.SH "ENVIRONMENT VARIABLES"
+.SS "OCK_USER_PIN"
+The token PIN can be specified via the environment variable OCK_USER_PIN.
+If nothing is set, pkcsicsf will prompt for the token PIN interactively.
+.PP
+.SS "OCK_SO_PIN"
+The security-officer PIN can be specified via the environment variable
+OCK_SO_PIN.  If nothing is set, pkcsicsf will prompt for the token PIN
+interactively.
+.PP
+.SS "PKCS11_USER_PIN (deprecated)"
+Same as OCK_USER_PIN. If set, OCK_USER_PIN is prefered over PKCS11_USER_PIN.
+This environment variable is deprecated.
```

Comment:
preferred

Reply:
ok. Maybe I should grep for "prefered" and fix it all... <grml>